internal/handler/job/tags/add.go

```diff
@@ -31,6 +35,18 @@ func Add(ctx context.Context, storage persistence.Storage, jobID string, tags []
 		return nil, fault.Wrap(err)
 	}
 
-	contextLogger.Info().Msg("Updated job tags")
+	contextLogger.Debug().Msg("Publishing event")
```

Comment:
Hm,  this "Publishing event" can only be related reading the next log line "`op` job tags". Wouldn't it make sense to include this information here as well, e.g., "Publishing event: `op` job tags."?